frontend/src/components/YearsDropdown.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import ListItemText from '@material-ui/core/ListItemText';
+import Select from '@material-ui/core/Select';
+
+const yearRanges = [
+  '0-5 years',
+  '6-10 years',
+  '11-15 years',
+  '16-20 years',
+  '20+ years',
+];
+
+// Add onChange to props validation
+YearDropdown.propTypes = {
+  onChange: PropTypes.func,
+};
+
+/**
+ * Create year dropdown component using yearRanges array
+ * @return {YearDropdown} YearDropdown component
+ * @param {Object} props
+ */
+export default function YearDropdown(props) {
```

Comment:
Any ways we can some how combine these *Dropdown() into a single function buy taking different label and inputs? This should make the code easier to extend with more fields, as I see these dropdowns are very similar

Reply:
I was initially trying to do this exact thing by taking different labels and inputs within, but problems arise when trying to get the state of each component on submit and the `id` of each dropdown would be the same

Reply:
Ok! (Since I am not familiar with React or JS, I don't know how to fix it. So...) I think it's OK to leave as is